test/debugger/test-debugger-client.js

```diff
@@ -168,7 +168,7 @@ function doTest(cb, done) {
 
       function tryConnect() {
         // Wait for some data before trying to connect
-        var c = new debug.Client();
+        const c = new debug.Client();
         console.error('>>> connecting...');
         c.connect(debug.port);
         c.on('break', function(brk) {
```

Comment:
Nit: While you're in here, maybe remove `brk` as a callback argument. It is unused.